site/content/Getting Started/_index.md

```diff
@@ -0,0 +1,139 @@
+* `http` specifies the HTTP [input format](/vhs/reference/#input-formats). This input format will extract HTTP requests and responses from the
+  captured TCP data streams.
+
+### `--output "json|stdout"`
+This portion of the command specifies the output portion of the data flow for this `vhs` session. A `vhs` session may
+have any number of outputs.
+
+In this case:
+* `json` specifies the JSON [output format](/vhs/reference/#output-formats). This output format will serialize the HTTP requests and responses into JSON.
+* `stdout` specifies the standard [output sink](/vhs/reference/#sinks). This sink will print the data to the console.
+
+### `--capture-response`
+This flag tells the TCP input source to capture two-way network traffic (requests and responses) instead of one-way
+(requests only).
+
+### `--address 0.0.0.0:1111`
+This flag specifies the address and port from which `vhs` will capture traffic, in the form `<IP address>:<port>`.
+
+### `--middleware ./testdata/http_middleware.bash`
+This flag specifies the middleware to run for this `vhs` session. Middleware enables users to modify or rewrite data
+as it passes through `vhs` from source to sink. The middleware specified here appends
+`" [[hijacked <HTTP_MESSAGE_TYPE>]]"` to the end of the http request or response body as it passes through `vhs`. More
+information on middleware can be found [here](/vhs/reference/#middleware).
+
+### `| jq -R "fromjson | .body"`
+This portion of the command pipes the output of `vhs` to `jq` for filtering and pretty printing. This functionality is
+external to `vhs`. You can find more information on `jq` [here](https://stedolan.github.io/jq/).
+
+### `2> /dev/null`
+Discards stderr to keep the demo output clean.
+
+## Common Use Cases
+
+### Capture live HTTP traffic to cloud storage
+`./vhs --input "tcp|http" --output "json|gzip|gcs" --address 0.0.0.0:80 --capture-response --gcs-bucket-name <some_bucket> --gcs-object-name <some_object> --flow-duration 2m`
+
+The above command will capture live HTTP traffic on port 80 for 2 minutes and save the captured data as gzipped JSON to
+Google Cloud Storage (GCS) in the specified bucket and object.
+
+### Generate HAR file from saved HTTP data
+`./vhs --input "gcs|gzip|json --output "har|stdout" --gcs-bucket-name <some_bucket> --gcs-object-name <some_object> --flow-duration 15s > harfile.json`
+
+The above command will "replay" saved HTTP data in gzipped JSON format from the specified GCS bucket and object and 
+produce an HTTP Archive ([HAR](http://www.softwareishard.com/blog/har-12-spec/)) file called `harfile.json` on the local
+filesystem.
+
+### Provide live HTTP metrics
+`./vhs --input "tcp|http" --address 0.0.0.0:80 --capture-response --prometheus-address 0.0.0.0:8888 --flow-duration 60m`
+
+The above command will capture live HTTP traffic on port 80 and calculate 
+[RED metrics](https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture/) for the captured
```

Comment:
Probably worth listing out the metrics + comments here.

Reply:
I've added a bit more of a description of the metrics provided as well as link to the reference section on metrics that describes all the metrics in much more detail.